recipes/openmm-xtb/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+CMAKE_FLAGS="${CMAKE_ARGS} -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
+
+
+rm -rf build || true
+
+
+CMAKE_FLAGS="${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release"
+CMAKE_FLAGS+=" -DPython3_EXECUTABLE=${PYTHON}"
+CMAKE_FLAGS+=" -DOPENMM_DIR=${PREFIX}"
+
+
+mkdir build && cd build
+cmake ${CMAKE_FLAGS} ${SRC_DIR}
+make -j$CPU_COUNT
+make -j$CPU_COUNT install PythonInstall
+
+# Copy tests to the share folder
+# Include test executables too
+TEST_DIR="${PREFIX}/share/${PKG_NAME}/tests/"
```

Comment:
Typically, we don't install tests. Why are you doing it here?

Reply:
@jaimergp this is copied from the build script you wrote for openmm-plumed.  Do you remember what the reason was?

Reply:
@xhochy is there a way to run the tests without copying them to the package?  I looked up a few other packages to see how they handled tests, but they did exactly the same thing as here.

According to the [conda-build documentation](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#test-section) you can add a `source_files` section to the recipe to make it copy files to the directory where the tests will be run.  I tried adding

```yaml
test:
  source_files:
    - python/tests/TestXtbForce.py
```

But when I then try to run it with `python TestXtbForce.py`, it says the file isn't there.

Reply:
This was handled this way in Plumed because they were only four files and it was easier that way. IIRC, the test files also had the prefix hardcoded and tests would fail otherwise. It was also the only way to test reliably whether the package was correctly built.

`files` and `source_files` will end up "packaging" the files too in the sense that they will be published as part of the `info/` directory, although maybe not part of the PREFIX, if that's what we are concerned with.

> But when I then try to run it with python TestXtbForce.py, it says the file isn't there.

Try `python python/tests/TestXtbForce.py`. They are copied with the relative paths untouched.

Reply:
The situation is the same here.  There are only two files (one C++ test and one Python test), both very small.  If `source_files` also adds it to the package, I don't see any advantage to doing it that way?